web/src/compose_setup.js

```diff
@@ -436,6 +447,7 @@ export function initialize() {
 
     $("textarea#compose-textarea").on("focus", () => {
         compose_recipient.update_placeholder_text();
+        compose_notifications.maybe_show_one_time_messages_fading_banner();
```

Comment:
I don't know what the cause is, but the first time that I click on a message to open compose in an interleaved view, this doesn't appear.

Reply:
So, the reason was `respond_to_message()` flow, first opens the compose box then adds fading.

Fixed it by adding a call to this `maybe_....` function in `complete_starting_tasks()` with a comment.